PWGCF/FemtoDream/FemtoDreamTrackSelection.h

```diff
@@ -425,6 +436,13 @@ bool FemtoDreamTrackSelection::isSelectedMinimal(T const& track)
   if (nRejectNotPropagatedTracks && std::abs(dca) > 1e3) {
     return false;
   }
+  if (nRejectITSHitandTOFMissing) {
+    TrackSelection* o2PhysicsTrackSelection = new TrackSelection(getGlobalTrackSelection());
+    ;
+    o2PhysicsTrackSelection->SetRequireHitsInITSLayers(1, {0, 1, 2, 3});
+    // Uncomment only when TOF timing is solved
+    // bool itsHit = o2PhysicsTrackSelection->IsSelected(track, TrackSelection::TrackCuts::kITSHits);
+  }
```

Comment:
Please, reconsider this
You cannot be creating and leaving there a track selection object per track